data/monster/Sorcerers/incredibly_old_witch.xml

```diff
@@ -1,83 +1,80 @@
-			<attribute key="areaEffect" value="blueshimmer" />
+		<attack name="outfit" interval="2000" chance="12" range="7" radius="4" target="1" monster="kongra" duration="60000">
+			<attribute key="areaEffect" value="teleport" />
+			<attribute key="shootEffect" value="suddendeath" />
 		</attack>
-		<attack name="outfit" interval="1000" chance="12" range="7" radius="4" monster="dog" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
+		<attack name="outfit" interval="2000" chance="12" range="7" radius="4" target="1" monster="dog" duration="60000">
+			<attribute key="areaEffect" value="teleport" />
+			<attribute key="shootEffect" value="suddendeath" />
 		</attack>
-		<attack name="outfit" interval="1000" chance="12" range="7" radius="4" monster="skunk" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
+		<attack name="outfit" interval="2000" chance="12" range="7" radius="4" target="1" monster="skunk" duration="60000">
+			<attribute key="areaEffect" value="teleport" />
+			<attribute key="shootEffect" value="suddendeath" />
 		</attack>
-		<attack name="outfit" interval="1000" chance="12" range="7" radius="4" monster="donkey" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
+		<attack name="outfit" interval="2000" chance="12" range="7" radius="4" target="1" monster="donkey" duration="60000">
+			<attribute key="areaEffect" value="teleport" />
+			<attribute key="shootEffect" value="suddendeath" />
 		</attack>
 	</attacks>
-	<defenses>
-		<defense name="outfit" interval="2000" chance="5" monster="rat" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="chicken" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="green frog" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="bug" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="pig" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="kongra" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="dog" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="skunk" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-		<defense name="outfit" interval="2000" chance="5" monster="donkey" duration="2000">
-			<attribute key="areaEffect" value="blueshimmer" />
-		</defense>
-	</defenses>
+	<defenses armor="199" defense="199" />
```

Comment:
this could be 0, cause you cant damage her anyway.

Reply:
just omit `armor` and `defense` entirely? Where does it come from if it's immune?

Reply:
> just omit `armor` and `defense` entirely? Where does it come from if it's immune?

copy&paste from deathslicer 🤣